app/src/config/registry.ts

```diff
@@ -48,15 +48,17 @@ export namespace Mainnet {
   //   supportedAddressTypes: ['ss58'],
   // }
 
-  // export const Moonbeam: Chain = {
-  //   uid: 'moonbeam',
-  //   name: 'Moonbeam',
-  //   logoURI: 'https://parachains.info/images/parachains/1716448660_moonbeam_logo.jpg',
-  //   chainId: 2004,
-  //   destinationFeeDOT: '', // To be confirmed
-  //   network: Network.Polkadot,
-  //   supportedAddressTypes: ['evm'],
-  // }
+  export const Moonbeam: Chain = {
+    uid: 'moonbeam',
+    name: 'Moonbeam',
+    logoURI: 'https://parachains.info/images/parachains/1716448660_moonbeam_logo.jpg',
```

Comment:
```suggestion
    logoURI: 'https://s2.coinmarketcap.com/static/img/coins/64x64/6836.png',
```
Can we prefer coinmarketcap? :)